configuration/csv-to-docs.md

```diff
@@ -1,32 +1,276 @@
+| health_center_1       | district_1                                         | false             | HC1  | 1544031155715           |
+| health_center_2       | district_2                                         | false             | HC2  | 1544031155715           |
+| health_center_3       | district_3                                         | false             | HC3  | 1544031155715           |
+
+The resulting doc structure would be:
+
+```
+{
+  "type": "health_center",
+  //Parent property with the _id from district_1 as the value.
+  "parent": "0c31056a-3a80-54dd-b136-46145d451a66",
+  "is_name_generated": "false",
+  "name": "HC3",
+  "external_id": "",
+  "notes": "",
+  "geolocation": "",
+  "reported_date": 1544031155715,
+  "_id": "45293356-353c-5eb1-9a41-baa3427b4f69"
+}
+```
 
-Lastly run `medic-conf --local upload-docs` to upload the converted docs into your local instance. `--local` can be replaced with an instance or URL. See [medic-conf](https://github.com/medic/medic-conf) for detailed instructions.
+## Creating CSV files for Users
 
-### Google Drive Authentication
 
-Medic-conf leverages google authentication to access google drive. You will need to create a client_secrets file named `.gdrive.secrets.json` and place it in the same directory where you want the CSVs to be fetched. [Token Creation](https://developers.google.com/identity/protocols/OAuth2InstalledApp)
+### Creating a new user with a new place
 
-Create the `.gdrive.secrets.json` file by downloading the `client_secrets.json` from google. You will need a CLIENT_ID, CLIENT_SECRET and REDIRECT_URL. You can find these pieces of information by going to the Developer Console, clicking your project --> APIs & auth --> credentials --> Download JSON. This will download the credentials but will need modified to be in this structure. 
+To create new users associated to new place and a new contact. Provide values for contact.name, place.name, and place.parent(can be existing place)
+
+```
+username,password,roles,name,phone,contact.name,place.c_prop,place.type,place.name,place.parent
+alice,Secret_1,district-admin,Alice Example,+123456789,Alice,p_val_a,health_center,alice area, district_uuid
+bob,Secret_1,district-admin,bob Example,+123456789,bob,p_val_a,health_center,bob area,disctrict_uuid
+```
+
+### Linking users to contacts created from csv-to-docs
+
+To create user accounts for contacts that are created while running csv-to-docs action follow these steps.
+
+1. Create a `users.csv` file in the `csv` folder with the rest of the csvs needed for `csv-to-docs` action.
+1. Add columns for username, password, roles, phone, contact, place, and any other additional fields you want to populate.
+1. Use the following query language as the header names: for contact `contact:person WHERE reference_id=COL_VAL and place place:GET _id OF place WHERE reference_id=COL_VAL`. This feature is also supported in the csv-to-docs csv files.
```

Comment:
Isn't this the same technology as described above? Shouldn't we just say that you can use the feature above to link, and then give a brief example instead of describing it over again?

Reply:
Yes, tried to clear this up.